README.md

```diff
@@ -1 +1,205 @@
-http://jgeewax.github.io/phoenix
+# [Phoenix](http://jgeewax.github.io/phoenix)
+
+## Getting Started
+
+Running Phoenix locally is very simple, you'll need `npm`, `bower` and `gulp` 
+installed.
+
+Clone the repository.
+
+```sh
+$ git clone git@github.com:jgeewax/phoenix.git
+```
+
+Install all the project dependencies
+
+```sh
+$ npm install && bower install
+```
+
+Then start the server
+
+```sh
+$ gulp serve
+```
+
+And you're good to go!
+
+## Developing a Plugin
+
+Phoenix is still in a very early stage, so the process for creating plugins is
+still very young and subject to change.
+
+### Create repo for your plugin
+
+**NOTE:** Currently we only support plugins hosted on public GitHub
+repositories.
+
+Your plugin name should be very specific to what it does, since we're using
+Angular 1.x modules for plugins we need to avoid naming collisions. For example,
+if your plugin is named `storage-browser` you should prefix any
+controller/view/services with that name. (`StorageBrowserCtrl`, etc.)
```

Comment:
Can we suggest a "." or "-" or something as a more namespace-ish separator, or does that not work in Angular land?